templates/hound.yml

```diff
@@ -1,17 +1,17 @@
 # See https://houndci.com/configuration for help.
 coffeescript:
-  # config_file: .coffeescript-style.json
+  config_file: .coffeescript-style.json
   enabled: true
 haml:
   # config_file: .haml-style.yml
   enabled: true
 javascript:
-  # config_file: .javascript-style.json
+  config_file: .javascript-style.json
   enabled: true
   # ignore_file: .javascript_ignore
 ruby:
-  # config_file: .ruby-style.yml
+  config_file: .ruby-style.yml
   enabled: true
 scss:
-  # config_file: .scss-style.yml
+  config_file: .scss-style.yml
```

Comment:
OK, cool. That matches the reasoning I have above for using the Rubocop default name.